plugins/modules/ec2_vpc_endpoint.py

```diff
@@ -29,6 +29,20 @@
     choices: [ "Interface", "Gateway", "GatewayLoadBalancer" ]
     type: str
     version_added: 1.5.0
+  vpc_endpoint_subnets:
+    description:
+      - The list of subnets to attach to the endpoint (Works only with Interface and GatewayLoadBalancer type endpoint).
```

Comment:
Can you please add a check for this combination of options to the module?  An example can be seen here:  https://github.com/ansible-collections/amazon.aws/blob/main/plugins/modules/ec2_instance.py#L1755-L1761

Reply:
Thanks, I made asked changes

Could you please review my PR?